bin/kmstool-enclave-cli/main.c

```diff
@@ -68,6 +75,8 @@ static struct aws_cli_option s_long_options[] = {
     {"aws-secret-access-key", AWS_CLI_OPTIONS_REQUIRED_ARGUMENT, NULL, 's'},
     {"aws-session-token", AWS_CLI_OPTIONS_REQUIRED_ARGUMENT, NULL, 't'},
     {"ciphertext", AWS_CLI_OPTIONS_REQUIRED_ARGUMENT, NULL, 'c'},
+    {"key-id", AWS_CLI_OPTIONS_REQUIRED_ARGUMENT, NULL, 'K'},
```

Comment:
may be small `k` for consistency?

Reply:
Since small `k` is already assigned to `aws-access-key-id`, I set it to large `K`, do I need to modify it?

Reply:
sorry, didn't notice it, yes, then let's keep it as `K`